roles/nodepool/files/etc/nodepool/elements/nodepool/cleanup.d/90-validate-dib-image

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+function reporter () {
+  message="$1"
+  shift
+  echo
+  echo "${message}"
+  for (( i=0; i<${#message}; i++ )); do
+    echo -n '-'
+  done
+  echo
+}
+
+function build_failure () {
+  reporter "Error encountered! Exiting build."
+  reporter "$1" >&2
+  exit 1
+}
+
+function check_python () {
+  reporter "Validating python installation."
+  if [[ ! $(python --version) == *"Python 2.7"* ]]; then
```

Comment:
Fun fact! The output of `python --version` is actually sent to stderr (I'm not sure why).

This can be seen by running the following:

```shell
$ python --version | cut -d ' ' -f2
Python 2.7.13
$ python --version 2>&1 | cut -d ' ' -f2
2.7.13
```

As such, we actually want this line to be:

```shell
  if [[ ! "$(python --version 2>&1)" == *"Python 2.7"* ]]; then
```

**NOTE:** without this change, the script will always fail on `check_python`.